### DISCUSSION TITLE
Email notifications silently fails

### DISCUSSION THREAD

**Discussion by prologic** [outline/outline#5759] [Self Hosting]
Hi Team, 👋

Just wondering how I can debug outbound emails from the outline backend? I'm not convinced this is working, although there are no errors in the logs.

What I've done so far to test this:

- Send an invite to a random email address I own/manage
- See if the email gets there
- Check the logs

So far nothing.

**Comment by tommoor** [comment]
The only thing you'll see in Outline's logs is `EmailTask running` and potentially a failure with an error if the SMTP connection cannot be made. If the mail can be placed on the SMTP server then you would need to use their interface to debug any deliverability issues.

**Reply by prologic** [reply]
Ahh I see it now! I just needed ot know what I was looking for 🤣

```
wiki_outline.1.wb2zc8g4323b@xxx    | {"error":"580B05B39C7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n","level":"error","message":"Error sending email to xxx","stack":"Error: 580B05B39C7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n"}
```

What do I need to configure or change to make this work correctly?

**Reply by tommoor** [reply]
Generally "wrong version number" means that the port you are trying to connect to doesn’t serve any TLS. You may need to connect to a different hostname or explicitly set the port, eg `SMTP_PORT=465`

**Reply by prologic** [reply]
Hmmm isn't port 587 the SMTP client port with STARTTLS? AFAIK SMTPS (465) is considered "legacy"? I'm using Poste.io as my mail server (https://poste.io/doc/getting-started) and have no issues with other clients using SMTP + STARTTLS on port 587

**Reply by tommoor** [reply]
You can also try setting the environment variable `SMTP_TLS_CIPHERS=SSLv3`

**Reply by prologic** [reply]
I actually already have this environment value set.

**Reply by prologic** [reply]
I've worked around this for now by using `SMTP_PORT=25` and `SMTP_SECURE=false`. I can't seem to make STARTTLS work properly with Outline. Bug maybe? 🤔

**Reply by prologic** [reply]
It's an internal network anyway, so I'm oaky with the work-around
